docs/data-ai/reference/mlmodel-design.md

```diff
@@ -12,6 +12,7 @@ date: "2022-01-01"
 # SME: Bijan Haney
 aliases:
   - /registry/advanced/mlmodel-design/
+  - /operate/reference/advanced-modules/mlmodel-design/
```

Comment:
Probably safe to not have this alias since it only existed for like 10 hours? :D Guess it doesn't hurt to keep it